src/particle.jl

```diff
@@ -64,6 +94,15 @@ particle_shape(::Type{<:PMuPlus}) = Track()
 particle_shape(::Type{<:PHadronShower}) = Cascade()
 particle_shape(::Type{<:PLightSabre}) = Track()
 
+
```

Comment:
Same comment here, the is_neutrino methods take a type as argument. Maybe also add a method similar to the particle_shape(particle) method that accepts a particle instead of a type?